lib/humboldt/cli.rb

```diff
@@ -60,7 +61,19 @@ def run_local
       end
       input_glob = File.join(options[:data_path], options[:input])
       hadoop_config_path = options[:hadoop_config] || default_hadoop_config_path
-      run_command('hadoop', 'jar', project_jar, '-conf', hadoop_config_path, job_config, input_glob, output_path, *options[:extra_hadoop_args])
+      hadoop_command = ['hadoop', 'jar', project_jar]
+      hadoop_config = ['-conf', hadoop_config_path]
+      job_args = [input_glob, output_path, *options[:extra_hadoop_args]]
+      command = hadoop_command.dup
+      if Rubydoop::VERSION > '2.0.0'
```

Comment:
I think the comparison should be `>=`, not `>`.

Also, comparing strings might theoretically give the wrong result. Perhaps something like this instead:
```ruby
if Gem::Version.new(Rubydoop::VERSION) >= Gem::Version.new('2.0.0')
```